Formula/o/opencilk.rb

```diff
@@ -0,0 +1,732 @@
+class Opencilk < Formula
+  desc "Task-parallel programming platform"
+  homepage "https://www.opencilk.org/"
+  url "https://github.com/OpenCilk/opencilk-project/archive/refs/tags/opencilk/v3.0.tar.gz"
+  version "19.1.7"
```

Comment:
Where does this version come from?

Reply:
This version is the version of LLVM that the OpenCilk compiler is based on.

I wasn't sure whether it's more appropriate to use the LLVM version or OpenCilk version here.  A large part of what OpenCilk provides is a modified version of Clang/LLVM, and other software can have complicated dependencies on compiler versions.  But I think I can change this version to match the OpenCilk version without breaking the formula.  Let me know what would make more sense.

Reply:
If upstream uses `v3.0`, so should we.